Installing-and-Configuring/OS/index.md

```diff
@@ -58,19 +58,27 @@ Your other partitions may be encrypted or not based on your preferences and this
+
+This option adds an extra layer of security. The TPM will only release the LUKS Disk Unlock key if the measurements match (again, LUKS header, firmware measurements and Disk Unlock Key passphrase releasing TPM VN stored Disk Unlock key, to be injected as an additional initrd file, decompressed by OS on initial boot step and used to unlock the LUKS container prior proceeding, if and only if everything is congruent). Let's remember that booting a default boot option also verifies /boot detached signed digest prior of offering to unlock encrypted storage with a Disk Unlock Key passphrase.
+
+If the disk is moved to a different system by the owner the LUKS Disk Recovery Key passphrase is used to unlock encrypted container from slot 0 to access the encrypted data.  This model is similar to the [Security Dongle](#security-dongle-with-luks-root) model.
+
+Generally, an attacker would not have the LUKS Disk Unlock key in key slot 1, because it was generated under Heads with a random 128 characters key once and then sealed into TPM NV memory space, to be unsealed when provided with proper measurements and Disk Unlock Key passphrase. To be able to access this content in TPM, or observe that secret traveling from TPM to memory, physical access would be required with the help of a TPM Genie or other physical attack.
+
+The attacker should neither have access to the LUKS Disk Recovery Key nor its passphrase stored under LUKS keyslot 0, because it was setuped at install time, and should only be prompted when defining a new boot default to nenew/change the Disk Unlock Key passphrase after Heads detects changed core /boot components prompts user to detach sign /boot digest (sha256sum). It will then prompt to define a new default boot, which offers to add a disk encryption to the TPM.
```

Comment:
The attacker should neither have access to the LUKS Disk Recovery Key (LUKS keyslot 0) nor its associated passphrase, because it was setuped at install time and should only be typed from the user when renewewing/changing the Disk Unlock Key passphrase when setting a new boot default after Heads detects changed core /boot components and prompts user to detach sign /boot digest (sha256sum).

Reply:
updated.  push in a minute.